main.go

```diff
@@ -29,14 +29,20 @@ func main() {
 	noDevPackages := flag.Bool("no-dev", false, "Do not check packages listed under require-dev")
 	updateCacheOnly := flag.Bool("update-cache", false, "Update the cache (other flags are ignored)")
 	help := flag.Bool("help", false, "Output help and version")
```

Comment:
But there is already `--help` flag out of the box, why do we even need another help attribute?

Reply:
Because wanting to know the version is different from wanting to know how to use the tool, as detailed in the original description of this PR.

Reply:
I'm not sure what you're meaning then - my understanding is that this _is_ how the `--help` flag works; are you saying that the `flags` package provides support for `--help` out of the box or something?

Reply:
Exactly, flags package does support `--help` flag out of the box.

Reply:
What I want to propose to do:
```go
flag.Usage = func() {
     fmt.Fprintf(os.Stderr, "%s, built at %s\n", version, date)
     flag.PrintDefaults()
}
```
Then we can remove the useless `help` check. What do you think?

Reply:
I think that is out of scope of this PR, and would be better done on another branch :)

(it makes sense to me though, if the resulting functionality is exactly the same)